subgraph/manifest/data/rinkeby.json

```diff
@@ -0,0 +1,6 @@
+{
+  "network": "rinkeby",
+  "address": "0x0381374d658b2c2e564e954219d9a3cfc6ae3fcb",
+  "startBlock": 7207670,
```

Comment:
just to double check is the correct block number?

Reply:
Yes, this is the one i deployed in Rinkeby for testing